src/main/java/no/unit/nva/doi/transformer/MainHandler.java

```diff
@@ -81,7 +87,8 @@ public void handleRequest(InputStream input, OutputStream output, Context contex
 
         try {
             UUID uuid = UUID.randomUUID();
-            Publication publication = converter.toPublication(dataciteResponse, uuid, owner);
+            URI publisherID = OrgNumberMapper.toCristinId(orgNumber);
+            Publication publication = converter.toPublication(dataciteResponse, uuid, owner, publisherID);
```

Comment:
is this still only a datacite response?

Reply:
No, not when Orestis is ready with his changes.